modules/solid_mechanics/doc/content/source/auxkernels/ReactionForceAux.md

```diff
@@ -0,0 +1,29 @@
+# ReactionForceAux
+
+Save the reaction force corresponding to the variable DOFs into this AuxVariable.
+
+!syntax parameters /AuxKernels/ReactionForceAux
+
+# Usage
+
+First declare a tagged vector to save the residual contribution, i.e.
+
+!listing /test/tests/plane_stress/3D_finite_tension_pull.i block=Problem
+
+Then, inform the solid mechanics Physics about the declared vector tag. The Physics will automatically accumulate the residual contribution into the tagged vector.
+
+!listing /test/tests/plane_stress/3D_finite_tension_pull.i block=Physics
+
+Next, use the ReactionForceAux to read the accumulated residual contribution into an AuxVariable, i.e.
+
+!listing /test/tests/plane_stress/3D_finite_tension_pull.i block=AuxKernels
+
+This would save the residual contribution in the $x$ direction into the AuxVariable named "react_x". Upon convergence of each time step, residual will be close to zero (up to the tolerances you set) everywhere in the domain except where strongly enforced constraints are applied. Examples of these constraints are Dirichlet boundary conditions and contact. The final, accumulated residual contribution on the constrained degrees of freedom can then be interpreted as the distributed "reaction force".
```

Comment:
```suggestion
In this example, this would save the residual contribution in the $x$ direction into the AuxVariable named "react_x". Upon convergence of each time step, the residual will be close to zero (up to the tolerances you set) everywhere in the domain except where strongly enforced constraints or other boundary conditions are applied. Examples of these constraints are Dirichlet boundary conditions and contact. The final, accumulated residual contribution on the constrained degrees of freedom can then be interpreted as the distributed "reaction force".
```

Reply:
They would be nonzero where there are traction BCs as well if you only accumulate to the tag from the Kernels, so I don't think we should say that this is only for strongly-enforced BCs.